exercise1.js

```diff
@@ -81,19 +124,34 @@ const nums = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
 // value in the nums array, and every 3rd number, i.e. 0, 3, 6,
 // and 9: the zeroth, third, sixth, and ninth values.
 
-//your code...
-
-
+var i;
+for (i = 0; i < nums.length; i=(i+3)) {
+        console.log(nums[i]);
+    }
+console.log("");
 
 // Problem 9:
 const foodArray = [ 'potatoes', 'tamales', 'lemon','strawberries','chocolate', 'pudding', {school : 'TEKcamp'} ];
 //access the value of the last element of the array and set it to a variable called school.  print the school variable to the console.
 
+var school = (foodArray[foodArray.length - 1]);
+console.log(school);
+console.log("");
+
 const adjectiveArray = [ 'salty', 'spicy', 'sour', 'sweet', 'rich','creamy','amazing'];
 // Using both the foodArray and the adjectiveArray, make a
 // "for" loop that console.log()'s a sentence for each
 // corresponding value in the arrays. Add the word "is" or "are" depending on if the food is singular or plural.  "Potatoes are salty", "Lemon is sour".
 
+var i;
+for (i = 0; i < foodArray.length-1; i++) {
+    var word = (foodArray[i][foodArray[i].length - 1] == 's' ? ' are ' : ' is ');
+    console.log(foodArray[i] + word + adjectiveArray[i] + ".");
+}
+console.log("");
+
+(foodArray)
+
```

Comment:
you didn't grab the value out of the object.  to do that, you can do `Object.values(foodArray[foodArray.length-1])[0]`